### ISSUE TITLE
Annoying info logs

### ISSUE THREAD

**Issue by nhbess** [CALFEM/calfem-python#63]
Is there a way to stop the annoying info logs? I'm running this in a loop, and the useful information I am logging gets lost among so much calfem stuff.

```
Info    : GMSH -> Python-module
Info    : Reading ...
Info    : Done reading ...
Info    : Meshing 1D...
Info    : [  0%] Meshing curve 1 (Nurb)
...
Info    : Done meshing 1D (Wall 0.00704885s, CPU 0s)
Info    : Meshing 2D...
Info    : Meshing surface 1 (Plane, Frontal-Delaunay)
Info    : Done meshing 2D (Wall 0.0078547s, CPU 0s)
Info    : 261 nodes 528 elements
Info    : Writing ...
Info    : Done writing ....
```

**Comment by jonaslindemann** [comment]
I will look into it. Some of that is from the GMSH Python module.

**Comment by jonaslindemann** [comment]
Fixed in upcoming 3.6.6
